Makefile

```diff
@@ -1,14 +1,25 @@
 
 .PHONY: all install install-ci clean dist info test version
 
+OS_UNAME := $(shell uname)
+
+ifeq ($(OS_UNAME), Darwin)
+	LIB_INCLUDE_PATH = -I./native_lib/third-party/apple/Clibsodium.xcframework/macos-arm64_x86_64/Headers/
+	HEADER_INCLUDE_PATH = -L./native_lib/third-party/apple/Clibsodium.xcframework/macos-arm64_x86_64/
+endif
+
 all: dist
 
 install:
+ifneq ($(OS_UNAME), Darwin)
 	sudo apt-get install -qq libsodium-dev
+endif
 
 install-ci:
+ifneq ($(OS_UNAME), Darwin)
```

Comment:
for CI, we probably don't need this, since we only build it on Ubuntu. At least for now.